integration/travis/scheduler_travis_config.edn

```diff
@@ -44,6 +44,15 @@
                                :enforce? true
                                :tokens-replenished-per-minute 600}
               :user-limit-per-m 1000000}
+ :plugins {
+           :job-submission-valiator {:batch-timeout-seconds 40
+                             :factory-fn "cook.demo-plugin/submission-factory"
+                             :arguments {:submit-url #config/env "COOK_DEMO_HOOKS_SUBMIT_URL"}}
+           :job-launch-filter {:age-out-last-seen-deadline-minutes 10
+                         :age-out-first-seen-deadline-minutes 600
+                         :age-out-seen-count 10
+                         :factory-fn "cook.demo-plugin/launch-factory"
+                         :arguments {:launch-url #config/env "COOK_DEMO_HOOKS_LAUNCH_URL"}}}
```

Comment:
Do we need `:arguments`? Or can plugins simply retrieve their configuration via the `mount`ed `config` `defstate`?

Reply:
Changed.